index.html

```diff
@@ -37,6 +37,51 @@
     </nav>
 
 
+    <main class="flex">
+        <div class="card">
+            <div class="flex">
+                <div>
+                    <h1> Meow in! </h1>
+                    <p> You know that feeling when you miss sunshine?
+                        Yep, <em> that </em> is how much we missed you. <strong> Welcome back</strong>.
+                    </p>
+                </div>
+                <img src="assets/singing-barista.svg" alt="logo" width="250" height="100" />
+            </div>
+            <form>
+                <label>
+                    <span class="mb-07 block"> Your email </span>
+                    <input class="login-input" type="text" name="email" required>
+                </label>
+
+                <div class="mt-07">
+                    <label for="passwordInput"> Memorable password (yep, that one) </label>
+
+
+                    <div class=" flex mt-07 mb-07">
+                        <input id="passwordInput" type="password" name="password"
+                            class="login-input login-input--border-left" required>
+                        <button type="button" class="orange-button"> <svg width="10" height="10"
+                                xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                                <!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
+                                <path
+                                    d="M226.5 92.9c14.3 42.9-.3 86.2-32.6 96.8s-70.1-15.6-84.4-58.5s.3-86.2 32.6-96.8s70.1 15.6 84.4 58.5zM100.4 198.6c18.9 32.4 14.3 70.1-10.2 84.1s-59.7-.9-78.5-33.3S-2.7 179.3 21.8 165.3s59.7 .9 78.5 33.3zM69.2 401.2C121.6 259.9 214.7 224 256 224s134.4 35.9 186.8 177.2c3.6 9.7 5.2 20.1 5.2 30.5v1.6c0 25.8-20.9 46.7-46.7 46.7c-11.5 0-22.9-1.4-34-4.2l-88-22c-15.3-3.8-31.3-3.8-46.6 0l-88 22c-11.1 2.8-22.5 4.2-34 4.2C84.9 480 64 459.1 64 433.3v-1.6c0-10.4 1.6-20.8 5.2-30.5zM421.8 282.7c-24.5-14-29.1-51.7-10.2-84.1s54-47.3 78.5-33.3s29.1 51.7 10.2 84.1s-54 47.3-78.5 33.3zM310.1 189.7c-32.3-10.6-46.9-53.9-32.6-96.8s52.1-69.1 84.4-58.5s46.9 53.9 32.6 96.8s-52.1 69.1-84.4 58.5z" />
+                            </svg>
+                            Show
+                        </button>
+                    </div>
+                </div>
+                <button type="submit" class="green-button"> Sign into your dashboard </button>
+            </form>
+            <div class="flex gap2">
+                <a href="https://www.smashingmagazine.com/auth/amnesia/#main"> Forgot your password?
```

Comment:
Let's add a blue color to the links

<img width="411" alt="image" src="https://user-images.githubusercontent.com/11327761/199676658-83282d7a-621d-438b-a47e-20031d66c719.png">